lib/maintenance_tasks/engine.rb

```diff
@@ -17,6 +17,12 @@ class Engine < ::Rails::Engine
     end
 
     config.after_initialize do
+      if MaintenanceTasks.error_handler.arity.equal?(1)
```

Comment:
I've moved the deprecation check into the `after_initialize`  block in the engine. Seems like a sensible place to put it?

Reply:
I think I would have overridden `error_handler=` to do that, that way we can't miss it.

Reply:
AH, sorry misunderstood what you meant initially. I'll update this to raise on the setter